model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/NamedTests.xtend

```diff
@@ -8,25 +8,387 @@ import org.eclipse.xtext.testing.validation.ValidationTestHelper
+            "REQ-MDL-001",
+            "REQ-MDL-003",
+            "REQ-TYPE-001",
+            "REQ-TYPE-002",
+            "REQ-TYPE-003",
+            "REQ-TYPE-004",
+            "REQ-TYPE-005",
+            "REQ-TYPE-006",
+            "REQ-TYPE-007",
+            "REQ-TYPE-008",
+            "REQ-TYPE-009",
+            "REQ-ENT-001",
+            "REQ-ENT-003"
+    ])
+    def void testNamingOfEntityIdentifier() {
+        '''
+         model modelTC021;
+
+         import judo::types;
+
+         type string SType(min-size = 0, max-size = 10);
+         type numeric Integer(precision = 9, scale = 0);
+
+         enum Enum {
+             a0 = 0;
+             b1 = 1;
+         }
+         entity Ent1 {
+             identifier Boolean xBool;
+             identifier SType xString;
+             identifier Integer xNumeric;
+             identifier Enum xEnum;
+             identifier Date xDate;
+             identifier Timestamp xTimestamp;
+             identifier Time xTime;
+         }
+        '''.parse => [
+            m | m.assertError(JsldslPackage::eINSTANCE.named, JslDslValidator.JAVA_BEAN_NAMING_ISSUE)
+        ]
+    }
+
+    @Test
+    @TestCase("TC021")
+    @Requirement(reqs = #[
+            "REQ-SYNT-001",
+            "REQ-SYNT-002",
+            "REQ-SYNT-003",
+            "REQ-SYNT-004",
+            "REQ-MDL-001",
+            "REQ-MDL-003",
+            "REQ-ENT-001",
+            "REQ-ENT-003",
+            "REQ-ENT-004",
+            "REQ-ENT-005",
+            "REQ-ENT-006"
+    ])
+    def void testNamingOfEntityRelation() {
+        '''
+         model modelTC021;
+
+         entity Ent1 {
+            relation Ent3 xEnt3;
+            relation Ent2 xEnt2 opposite xEnt1;
+            relation Ent4 xEnt4 opposite-add Ent4;
+
+         }
+         entity Ent2 {
+            relation Ent1 xEnt1 opposite xEnt2;
+         }
+         entity Ent3 {
+         }
+         entity Ent4 {
+         }
+        '''.parse => [
+            m | m.assertError(JsldslPackage::eINSTANCE.named, JslDslValidator.JAVA_BEAN_NAMING_ISSUE)
+        ]
+    }
+
+    @Test
+    @TestCase("TC021")
+    @Requirement(reqs = #[
+            "REQ-SYNT-001",
+            "REQ-SYNT-002",
+            "REQ-SYNT-003",
+            "REQ-SYNT-004",
+            "REQ-MDL-001",
+            "REQ-MDL-003",
+            "REQ-TYPE-001",
+            "REQ-TYPE-002",
+            "REQ-TYPE-003",
+            "REQ-TYPE-004",
+            "REQ-TYPE-005",
+            "REQ-TYPE-006",
+            "REQ-TYPE-007",
+            "REQ-TYPE-008",
+            "REQ-TYPE-009",
+            "REQ-ENT-001",
+            "REQ-ENT-008"
+    ])
+    def void testNamingOfDerived() {
+        '''
+         model modelTC021;
+
+         import judo::types;
+
+         type string SType(min-size = 0, max-size = 10);
+         type numeric Integer(precision = 9, scale = 0);
+
+         enum Enum {
+              a0 = 0;
+              b1 = 1;
+         }
```

Comment:
Wher the multiple errors are tested? It only match for one of an error

Reply:
https://github.com/BlackBeltTechnology/judo-meta-jsl/pull/140/commits/c21043a6bddf93d2553d733773942aa934874006